### ISSUE TITLE
"←FAILED" message is printed for passing tests

### ISSUE THREAD

**Issue by asmundstavdahl** [asmundstavdahl/astest-dart#1]
```
00:00 +1: test/simple_node_test_test.dart: ...      Given  node = new SimpleNode(); ←FAILED        
00:00 +2: test/simple_node_test_test.dart: ...      Given  node = new SimpleNode(); ←FAILED
00:00 +3: test/simple_node_test_test.dart: ...      Given  node = new SimpleNode(); ←FAILED
00:00 +4: test/simple_node_test_test.dart: ...      Given  node = new SimpleNode(); ←FAILED
00:00 +5: test/simple_node_test_test.dart: ...      Given  node = new SimpleNode(); ←FAILED
00:00 +5: test/simple_node_test_test.dart: ...      Given  testValue = "A new value."; ←FAILED
00:00 +6: test/simple_node_test_test.dart: ...      Given  testValue = "A new value."; ←FAILED
00:00 +6: test/simple_node_test_test.dart: ...      Given  initialValue = node.value; ←FAILED
00:00 +7: test/simple_node_test_test.dart: ...      Given  initialValue = node.value; ←FAILED
00:00 +7: test/simple_node_test_test.dart: ...      Given  node.value = testValue; ←FAILED
00:00 +8: test/simple_node_test_test.dart: ...      Given  node.value = testValue; ←FAILED
00:00 +8: ... = testValue;
      Given  expect(node.value, isNot(initialValue)); ←FAILED
00:00 +9: ... = testValue;
      Given  expect(node.value, isNot(initialValue)); ←FAILED
00:00 +9: ... isNot(initialValue));
      Given  expect(node.value, equals(testValue)); ←FAILED
00:00 +10: ... isNot(initialValue));
      Given  expect(node.value, equals(testValue)); ←FAILED
00:00 +10: test/simple_node_test_test.dart: ...      Given  testValue2 = 42; ←FAILED
00:00 +11: test/simple_node_test_test.dart: ...      Given  testValue2 = 42; ←FAILED
00:00 +11: test/simple_node_test_test.dart: ...      Given  initialValue2 = node.value; ←FAILED
00:00 +12: test/simple_node_test_test.dart: ...      Given  initialValue2 = node.value; ←FAILED
00:00 +12: test/simple_node_test_test.dart: ...      When   node.value = testValue2; ←FAILED
00:00 +13: test/simple_node_test_test.dart: ...      When   node.value = testValue2; ←FAILED
00:00 +13: ... = testValue2;
      Then   expect(node.value, isNot(initialValue2)); ←FAILED
00:00 +14: ... = testValue2;
      Then   expect(node.value, isNot(initialValue2)); ←FAILED
00:00 +14: ...      Then   expect(node.value, equals(testValue2.toString())); ←FAILED
00:00 +15: ...      Then   expect(node.value, equals(testValue2.toString())); ←FAILED
00:00 +15: test/simple_node_test_test.dart: ...      Given  node = new SimpleNode(); ←FAILED
00:00 +16: test/simple_node_test_test.dart: ...      Given  node = new SimpleNode(); ←FAILED
00:00 +16: test/simple_node_test_test.dart: ... = "The value of the child node."; ←FAILED
00:00 +17: test/simple_node_test_test.dart: ... = "The value of the child node."; ←FAILED
00:00 +17: test/simple_node_test_test.dart: ...     Given  childKey = "name-of-the-child"; ←FAILED
00:00 +18: test/simple_node_test_test.dart: ...     Given  childKey = "name-of-the-child"; ←FAILED
00:00 +18: test/simple_node_test_test.dart: ... = new SimpleNode()..value = testValue; ←FAILED
00:00 +19: test/simple_node_test_test.dart: ... = new SimpleNode()..value = testValue; ←FAILED
00:00 +19: test/simple_node_test_test.dart: ...      When   node[childKey] = childNode; ←FAILED
00:00 +20: test/simple_node_test_test.dart: ...      When   node[childKey] = childNode; ←FAILED
00:00 +20: ... = childNode;
      Then   expect(node[childKey], equals(childNode)); ←FAILED
00:00 +21: ... = childNode;
      Then   expect(node[childKey], equals(childNode)); ←FAILED
00:00 +21: test/simple_node_test_test.dart: ...      Given  node = new SimpleNode(); ←FAILED
00:00 +22: test/simple_node_test_test.dart: ...      Given  node = new SimpleNode(); ←FAILED
00:00 +22: test/simple_node_test_test.dart: ...   Then   expect(node["sdasdasd"], isNull); ←FAILED
00:00 +23: test/simple_node_test_test.dart: ...   Then   expect(node["sdasdasd"], isNull); ←FAILED
00:00 +23: test/simple_node_test_test.dart: ...      Given  node = new SimpleNode(); ←FAILED
00:00 +24: test/simple_node_test_test.dart: ...      Given  node = new SimpleNode(); ←FAILED
00:00 +24: test/simple_node_test_test.dart: ... = new SimpleNode()..value="Node #1"; ←FAILED
00:00 +25: test/simple_node_test_test.dart: ... = new SimpleNode()..value="Node #1"; ←FAILED
00:00 +25: ... #1";
      Given  expect(node["key"].value, equals("Node #1")); ←FAILED
00:00 +26: ... #1";
      Given  expect(node["key"].value, equals("Node #1")); ←FAILED
00:00 +26: test/simple_node_test_test.dart: ... = new SimpleNode()..value="Node #2"; ←FAILED
00:00 +27: test/simple_node_test_test.dart: ... = new SimpleNode()..value="Node #2"; ←FAILED
00:00 +27: ... #2";
      Then   expect(node["key"].value, equals("Node #2")); ←FAILED
00:00 +28: ... #2";
      Then   expect(node["key"].value, equals("Node #2")); ←FAILED
00:00 +28: All tests passed!
```

**Comment by asmundstavdahl** [comment]
Caused by newlines in the test's names when scenarios/features contain more than one line.

**Comment by asmundstavdahl** [comment]
Fixed in v1.2.1 cf10600458aeeb7168fcfce00e209da5b08591f4.
